source/NPC.cpp

```diff
@@ -203,6 +212,29 @@ void NPC::Save(DataWriter &out) const
 		if(mustAccompany)
 			out.Write("accompany");
 		
+		// Only save out spawn conditions if they have yet to be evaluated.
+		// This is so that if a player quits the game and returns, NPCs that
+		// were spawned do not then become despawned because they no longer
+		// pass the spawn conditions.
+		if(!toSpawn.IsEmpty() && !checkedSpawnConditions)
+		{
+			out.Write("to", "spawn");
+			out.BeginChild();
+			{
+				toSpawn.Save(out);
+			}
+			out.EndChild();
```

Comment:
This syntax is copy-pasted from MissionAction.cpp, but I've noticed that not all files use this additional indentation + curly braces for `out.BeginChild(); ... out.EndChild();`. Might be worth standardizing.

Reply:
Yes, sounds appropriate for the style guide.